config/sync/user.role.site_editor.yml

```diff
@@ -20,6 +20,7 @@ dependencies:
     - node.type.stanford_person
     - node.type.stanford_policy
     - node.type.stanford_publication
+    - taxonomy.vocabulary.basic_page_filters
```

Comment:
@mdyoung3 we decided only Site Managers should have access for filtered views. That is what we did for Opportunities. Let's stick with that.